modules/navier_stokes/test/tests/finite_volume/pins/channel-flow/heated/tests

```diff
@@ -19,7 +19,7 @@
       input = 2d-rc-heated.i
       exodiff = rc-heated-rz.e
       method = "!dbg"
-      abs_zero = 1e-9
+      abs_zero = 2e-9
```

Comment:
Why was this necessary? What is different compared to the previous implementation?